application/controllers/SurveyAdministrationController.php

```diff
@@ -1945,7 +1945,7 @@ public function actionRendersidemenulink($surveyid, $subaction)
         $aData['action'] = $menuEntry->action;
         $aData['entryData'] = $menuEntry->attributes;
         $aData['dateformatdetails'] = getDateFormatData(Yii::app()->session['dateformat']);
-        $aData['subaction'] = $menuEntry->title;
+        $aData['subaction'] = gT($menuEntry->title);
```

Comment:
Came from model , must not translate if [$menuEntry->language](https://github.com/LimeSurvey/LimeSurvey/blob/master/application/models/SurveymenuEntries.php#L21) == admin language

Must translate only if $menuEntry->language is en (or [en-GB](https://github.com/LimeSurvey/LimeSurvey/blob/dfa8c7d07cbfc6f2987dbe248afe4824092df8be/application/core/LsDefaultDataSets.php#L39) ) .